openapi-operation-validator/src/main/java/org/openapi4j/operation/validator/model/impl/Body.java

```diff
@@ -61,13 +62,13 @@ public static Body from(JsonNode body) {
   }
 
   /**
-   * Constructs a body from the given map tree model.
+   * Constructs a body from the given Java Object.
    * This is the preferred way to build a body wrapper.
    *
    * @param body the given map tree model (JSON, XML, ...)
    * @return The constructed body.
    */
-  public static Body from(Map<String, Object> body) {
+  public static Body from(Object body) {
```

Comment:
Java doc should be updated to reflect the changes (look for 'map')

Reply:
I tidied it up - although I'm not sure what is best for the JavaDoc. It seems that TreeUtil.json.convertValue method can take a wide variety of inputs. I am also not clear on XML handling and why that is mentioned in the doc.